pom.client.xml

```diff
@@ -333,6 +333,31 @@
               <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
             </configuration>
           </execution>
+          <execution>
+            <id>check</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <rule>
+                  <element>BUNDLE</element>
+                  <limits>
+                    <limit>
+                      <counter>LINE</counter>
+                      <value>COVEREDRATIO</value>
+                      <minimum>0.18</minimum>
+                    </limit>
+                    <limit>
+                      <counter>BRANCH</counter>
+                      <value>COVEREDRATIO</value>
+                      <minimum>0.09</minimum>
```

Comment:
Is it worth extracting these two minimum values out to properties so that child poms may overwrite this (in the future when we start setting a valid minimum and libraries need to opt-out for a valid reason)?

Reply:
Yes, updated this to use properties and also customized the threshold for a couple of storage modules that had really low coverage to:
- demonstrate how to customize the threshold per module
- increase the base threshold to a much higher number than 9%